jax/linear_util.py

```diff
@@ -122,7 +122,7 @@ def __init__(self, f, transforms, stores, params):
   def __name__(self):
     return getattr(self.f, '__name__', '<unnamed wrapped function>')
 
-  def wrap(self, gen, gen_static_args, out_store):
+  def wrap(self, gen, gen_static_args, out_store) -> 'WrappedFun':
```

Comment:
Why is this return type in quotes?

Reply:
I guess it's related to being the enclosing class?

Reply:
Yes, the type name is not yet in scope, so you have to wrap it as a string.